modules/ROOT/pages/websockets/websocket-connection-management-components.adoc

```diff
@@ -0,0 +1,65 @@
+= About Websocket Connection Management Components
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: connectors, ws, wss, websocket
+
+== About Connection Management Components
+
+These components provide tools for connection management for both server and client side operations. All of these operations are available for both Inbound and Outbound sockets.
+
+=== On Socket Closed
+
+This component's owner flow will be triggered each time a connection is closed. This is useful to perform cleanup operations such as logging or updating the application's state.
+
+
+[source,xml,linenums]
+----
+<flow name="onConnectionClosed">
+    <websocket:on-connection-closed ​path​=​"/ws/*" ​config-ref​=​"ws"​ />
+    <logger level="INFO" message="No more messages please: you shall not pass." />
+</flow>
+----
+
+Once this flow is invoked, no more messages can be processed because the socket has already been closed.
+
+=== Subscribe Groups
+
+This operation allows a Websocket to subscribe to one or more groups. It can be invoked on the same socket as many times as necessary, each invokation being additive over the previous ones. Repeated groups will be ignored.
+
+[source,xml,linenums]
+----
+<flow name="subscribeToGroup">
+ <websocket:subscribe-groups ​socketId​=​"#[attributes.socketId]" groups​=​"#[['MARVEL', 'DC']]"​ />
+</flow>
+----
+
+This is especially useful for broadcast operations in which a message can be delivered to one or more sockets associated or subscribed to a specific group (or groups).
```

Comment:
This should reference the openOutboundSocket operation and the InboundMessageListener which also allow to set the groups right when the connection is stablished.

I think that everything `groups` related is better served as a specific section about broadcasting and segregation in which these are explained in the context of that use case